libs/s25main/Loader.cpp

```diff
@@ -512,6 +521,9 @@ bool Loader::LoadFilesAtGame(const std::string& mapGfxPath, bool isWinterGFX, co
     if(!LoadResources({"wine_bobs"}))
         return false;
 
+    if(!LoadResources({"leather_bobs"}))
```

Comment:
Is it possible to condition those on the enabled addons? Feels wasteful but maybe the remaining code can't handle those to not be present?

The idea was also to integrate those with the "regular" files and have them be loaded automatically if the addon is enable (see the above TODO and `initResourceFolders` (addon argument)). Do you see any sensible archive for that?